### ISSUE TITLE
devtools::install_github does not break/error out even when installation failed on windows

### ISSUE THREAD

**Issue by ajdamico** [hannesmuehleisen/MonetDBLite#5]
i did a clean install_github and get this  :/

> library(DBI)
> library(MonetDB.R)
> library(MonetDBLite)
>  dbdir <- tempdir()
> con <- dbConnect(MonetDB.R::MonetDBLite(), dbdir)
Error in .Call("monetdb_startup_R", dir, quiet, getOption("monetdb.squential",  : 
  "monetdb_startup_R" not available for .Call() for package "libmonetdb5"
> sessionInfo()
R version 3.3.0 (2016-05-03)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 10240)

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] MonetDBLite_0.3.0 MonetDB.R_1.0.1   DBI_0.4-1        

loaded via a namespace (and not attached):
[1] codetools_0.2-14 digest_0.6.9

**Comment by hannesmuehleisen** [comment]
need full output

**Comment by ajdamico** [comment]
> devtools::install_github("hannesmuehleisen/MonetDBLite")
Skipping install for github remote, the SHA1 (9fa3f27a) has not changed
since last install.
  Use `force = TRUE` to force installation
> devtools::install_github("hannesmuehleisen/MonetDBLite",force=T)
Downloading GitHub repo hannesmuehleisen/MonetDBLite@master
from URL
https://api.github.com/repos/hannesmuehleisen/MonetDBLite/zipball/master
Installing MonetDBLite
"C:/PROGRA~1/R/R-33~1.0/bin/x64/R" --no-site-file --no-environ --no-save
--no-restore --quiet CMD INSTALL  \

"C:/Users/anthonyd/AppData/Local/Temp/RtmpgpkDc2/devtools17887d5b4785/hannesmuehleisen-MonetDBLite-9fa3f27"
\
  --library="C:/Users/anthonyd/Documents/R/win-library/3.3" --install-tests

* installing *source* package 'MonetDBLite' ...
src/acout.in
src/buildtools/conf/Makefile.in
src/buildtools/Makefile.in
src/common/Makefile.in
src/common/options/Makefile.in
src/common/stream/Makefile.in
src/common/stream/monetdb-stream.pc.in
src/common/utils/Makefile.in
src/embedded/Makefile.in
src/embedded/windows/monetdb_config.h.in
src/gdk/Makefile.in
src/gdk/monetdb-gdk.pc.in
src/Makefile.in
src/mal/Makefile.in
src/mal/mal/Makefile.in
src/mal/modules/atoms/Makefile.in
src/mal/modules/kernel/Makefile.in
src/mal/modules/Makefile.in
src/mal/modules/mal/Makefile.in
src/mal/optimizer/Makefile.in
src/mal/scheduler/Makefile.in
src/mal/tools/Makefile.in
src/mal/tools/monetdb5.pc.in
src/monetdb_config.h.in
src/sql/backends/Makefile.in
src/sql/backends/monet5/generator/Makefile.in
src/sql/backends/monet5/Makefile.in
src/sql/common/Makefile.in
src/sql/include/Makefile.in
src/sql/Makefile.in
src/sql/scripts/Makefile.in
src/sql/server/Makefile.in
src/sql/storage/bat/Makefile.in
src/sql/storage/Makefile.in
> download.file('http://dev.monetdb.org/Assets/R/misc/msvcr100-64.dll',
'src/embedded/windows/msvcr100.dll', mode='wb')
trying URL 'http://dev.monetdb.org/Assets/R/misc/msvcr100-64.dll'
Content type 'application/octet-stream' length 829264 bytes (809 KB)
==================================================
downloaded 809 KB

>
>
> setwd('src/embedded/windows/'); tf <- tempfile(); download.file('
http://dev.monetdb.org/Assets/R/misc/pcre-8.37.zip', tf, mode='wb');
unzip(tf)
trying URL 'http://dev.monetdb.org/Assets/R/misc/pcre-8.37.zip'
Content type 'application/zip' length 174489 bytes (170 KB)
==================================================
downloaded 170 KB

>
>
CDPATH="${ZSH_VERSION+.};" && cd . &&  -I buildtools/conf
-I: not found
make: ***
[c:/Users/anthonyd/AppData/Local/Temp/RtmpgpkDc2/devtools17887d5b4785/hannesmuehleisen-MonetDBLite-9fa3f27/src/aclocal.m4]
Error 127
** libs
Warning: this package has a non-empty 'configure.win' file,
so building only the main architecture

installing to
C:/Users/anthonyd/Documents/R/win-library/3.3/MonetDBLite/libs/x64
** R
** tests
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
* DONE (MonetDBLite)
>

**Comment by hannesmuehleisen** [comment]
yeah stuff missing from your PATH, needs to have c:/Rtools/bin and c:/Rtools/gcc-…/bin

**Comment by ajdamico** [comment]
that's not really relevant?  if the package did not install properly, the
install process should break

**Comment by hannesmuehleisen** [comment]
correct it should fail. open issue plz. it should also try to get those paths correct.

**Comment by ajdamico** [comment]
aye aye

**Comment by hannesmuehleisen** [comment]
also set $PATH correctly and also check this on Unix systems

**Comment by hannesmuehleisen** [comment]
added a check to see whether the library was created https://github.com/hannesmuehleisen/MonetDBLite/blob/master/configure.win#L32

**Comment by ajdamico** [comment]
look at you

On Mon, May 23, 2016 at 10:45 AM, Hannes Mühleisen <notifications@github.com
